pkg/manager/infosync/info.go

```diff
@@ -160,15 +160,21 @@ func (is *InfoSyncer) getTopologyInfo(cfg *config.Config) (*TopologyInfo, error)
 		s = ""
 	}
 	dir := path.Dir(s)
-	ip := sys.GetLocalIP()
-	_, port, err := net.SplitHostPort(cfg.Proxy.Addr)
+	ip, port, err := net.SplitHostPort(cfg.Proxy.Addr)
 	if err != nil {
 		return nil, errors.WithStack(err)
 	}
 	_, statusPort, err := net.SplitHostPort(cfg.API.Addr)
 	if err != nil {
 		return nil, errors.WithStack(err)
 	}
+	// reporting a non unicast IP makes no sense
+	// try to find one
+	if !net.ParseIP(ip).IsGlobalUnicast() {
```

Comment:
`net.ParseIP(ip).IsGlobalUnicast()` returns false for both `127.0.0.1` and `0.0.0.0`, does it make any difference?

Reply:
Mistake. I thought that loopback is `GlobalUnicast`, however it is only local unicast...

I've changed to only include `unspecified` or `multicast` IP.

Reply:
Add tests to make sure it's expected.